src/components/Layout.js

```diff
@@ -1,10 +1,18 @@
+import Footer from './Footer';
 import Header from './Header';
 
 export default function Layout({children}) {
 	return (
 		<>
 			<Header />
-			<main>{children}</main>
+			<main
+				style={{
+					paddingBottom: '40px',
+				}}
```

Comment:
Please do not use an inline-style here. This could be part of your `GlobalStyles`.

Reply:
Ok erledigt .